packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -251,6 +251,26 @@ class _PagePosition extends ScrollPositionWithSingleContext implements PageMetri
   final int initialPage;
   double _pageToUseOnStartup;
 
+  /// If [pixels] isn't set by [applyViewportDimension] before [dispose] is
+  /// called, this could throw an assert as [pixels] will be set to null.
+  ///
+  /// With [Tab]s, this happens when there are nested [TabBarView]s and there
+  /// is an attempt to warp over the nested tab to a tab adjacent to it.
+  ///
+  /// This flag will be set to true once the dimensions have been established
+  /// and [pixels] is set.
+  bool isInitialPixelsValueSet = false;
+
+  @override
+  void dispose() {
+    // Sets `pixels` to a non-null value before `ScrollPosition.dispose` is
```

Comment:
can you add a TODO and link to the issue and note once the issue is fix we can remove this workaround

Reply:
Done!